### ISSUE TITLE
No icon preview inside admin

### ISSUE THREAD

**Issue by hacknug** [7studio/acf-svg-icon#2]
Is the icon preview supposed to show up automatically?

I've managed to add titles and remove some unneeded symbols but Select2 only shows the icon's title without any kind of preview. `data-file_url` on the `<select>` element points to the right location of my sprite.

Here's a preview of my SVG file in case it helps:

```xml
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <symbol viewBox="0 0 75 75" id="euro" xmlns="http://www.w3.org/2000/svg">
        <path d="M11 41.2h7c-.1-1.3-.2-2.6-.2-3.8 0-1.1.1-2.2.2-3.5h-7V27h8.3c1.8-5.9 5.6-11 10.7-14.4 5-3.3 10.8-5.1 16.7-5.1 6.2 0 11.2 1.4 15.1 4.2l-3.5 6.8c-3.4-2.3-7.5-3.5-11.6-3.3-3.9 0-7.7 1-11.1 3-3.3 2-5.9 5.1-7.3 8.8h25.9l-3.5 6.9h-24c-.2 1.2-.3 2.4-.3 3.5 0 1.3.1 2.5.3 3.8h21l-3.5 7.1H28.5c1.4 3.6 3.9 6.6 7.2 8.6 3.3 2 7 3 10.8 2.9 5.6 0 10-1.6 13.2-4.8l4.3 6.2c-4.9 4.2-11.1 6.4-17.5 6.3-5.8 0-11.6-1.7-16.4-5-5-3.4-8.8-8.4-10.6-14.2H11v-7.1z" />
    </symbol>
    <symbol viewBox="0 0 75 75" id="yuan" xmlns="http://www.w3.org/2000/svg">
        <path d="M58.1 53.1H42.5v14.4h-9V53.1H17.9v-6.7h15.6V39H17.9v-6.6h10.8l-15.8-25h10.6L38 31.7 51.8 7.5h10.4L47 32.5h11v6.6H42.5v7.4h15.6v6.6z" />
    </symbol>
</svg>
```

**Comment by 7studio** [comment]
Hi @hacknug,

Yes, the icon preview is supposed to show up automatically :sweat_smile:

After few researches, it seems that `acf.select2.version` doesn't exist anymore in the new ACF Pro versions… Unfortunately, the plugin is broken now :worried:

I will fix this issue during the day.

Thank you for mentioning this issue :thumbsup:

**Comment by hacknug** [comment]
Cool. Let me know if I can help!

**Comment by 7studio** [comment]
Which version of ACF Pro do you use?

**Comment by hacknug** [comment]
That would be ACF Pro Version 5.7.3

**Comment by 7studio** [comment]
Hi @hacknug,
Can you try the fix with the help of the [`fix/select2-version-checking`](https://github.com/7studio/acf-svg-icon/tree/fix/select2-version-checking) branch please?

If it's OK for you, I will merge the branch :wink:

**Comment by hacknug** [comment]
Working perfectly fine. Thank you!

Maybe you can also tag a release besides merging the changes in?

**Comment by 7studio** [comment]
Hi @hacknug,

I merged the branch and I created a 1.0.1 release :smile:
